pages.de/common/borg.md

```diff
@@ -12,7 +12,7 @@
 
 `borg create --progress {{/pfad/zum/repo_verzeichnis}}::{{Montag}} {{/pfad/zum/quell_verzeichnis}}`
 
--  Alle Archive in einem Repository auflisten
+-  Alle Archive in einem Repository auflisten:
```

Comment:
Double space
```suggestion
- Alle Archive in einem Repository auflisten:
```

Reply:
@sbrl I noticed that as well - please see my comment above https://github.com/tldr-pages/tldr/pull/5214#issuecomment-770426912